.php_cs.cache

```diff
@@ -0,0 +1 @@
et":"newest"},"php_unit_mock":{"target":"newest"},"php_unit_mock_short_will_return":true,"php_unit_no_expectation_annotation":{"target":"newest"},"php_unit_ordered_covers":true,"php_unit_test_annotation":{"style":"prefix"},"php_unit_test_case_static_method_calls":{"call_type":"self"},"phpdoc_align":{"align":"vertical"},"phpdoc_no_useless_inheritdoc":true,"phpdoc_scalar":true,"phpdoc_separation":true,"phpdoc_single_line_var_spacing":true,"phpdoc_trim":true,"phpdoc_trim_consecutive_blank_line_separation":true,"phpdoc_types":true,"phpdoc_types_order":{"null_adjustment":"always_last","sort_algorithm":"none"},"phpdoc_var_without_name":true,"single_trait_insert_per_statement":true,"standardize_not_equals":true,"ternary_to_null_coalescing":true,"void_return":true},"hashes":{"src\/CachingStream.php":4124317593,"src\/ServerRequest.php":518813568,"src\/HttpFactory.php":2512390893,"src\/StreamWrapper.php":3417669187,"src\/UriResolver.php":2935615294,"src\/Utils.php":2252714247,"src\/Header.php":634796244,"src\/LimitStream.php":1321475186,"src\/Request.php":393656417,"src\/MimeType.php":1410430538,"src\/PumpStream.php":464940055,"src\/DroppingStream.php":1592685613,"src\/StreamDecoratorTrait.php":3076760683,"src\/UriNormalizer.php":4050008279,"src\/AppendStream.php":3974334693,"src\/Exception\/MalformedUriException.php":2085549722,"src\/MultipartStream.php":3140651942,"src\/Rfc7230.php":992502012,"src\/Stream.php":2384469106,"src\/Query.php":3235602399,"src\/MessageTrait.php":2633105155,"src\/LazyOpenStream.php":1681519827,"src\/NoSeekStream.php":87188491,"src\/Response.php":437588914,"src\/BufferStream.php":2550150489,"src\/Uri.php":1490285058,"src\/InflateStream.php":888470643,"src\/Message.php":3561159015,"src\/FnStream.php":1050820504,"src\/UploadedFile.php":1882955766,"tests\/UriNormalizerTest.php":2702753674,"tests\/PumpStreamTest.php":810496326,"tests\/FnStreamTest.php":3023696406,"tests\/ReadSeekOnlyStream.php":2239940617,"tests\/StreamTest.php":3234337377,"tests\/AppendStreamTest.php":3613868310,"tests\/InflateStreamTest.php":2400745463,"tests\/ResponseTest.php":3210667136,"tests\/DroppingStreamTest.php":4096510809,"tests\/CachingStreamTest.php":3686162664,"tests\/Integration\/ServerRequestFromGlobalsTest.php":4090171619,"tests\/Integration\/server.php":819196779,"tests\/UriTest.php":1635324753,"tests\/UploadedFileTest.php":704018714,"tests\/MultipartStreamTest.php":3554036590,"tests\/RequestTest.php":101334953,"tests\/MimeTypeTest.php":1109422250,"tests\/MessageTest.php":650906072,"tests\/StreamDecoratorTraitTest.php":1056619207,"tests\/HasToString.php":472560092,"tests\/UriResolverTest.php":3187700976,"tests\/LimitStreamTest.php":561003037,"tests\/NoSeekStreamTest.php":3031719965,"tests\/BufferStreamTest.php":4089247971,"tests\/UtilsTest.php":1642749182,"tests\/QueryTest.php":1370167524,"tests\/LazyOpenStreamTest.php":1225196132,"tests\/ServerRequestTest.php":4098073084,"tests\/StreamWrapperTest.php":3770959201,"tests\/HeaderTest.php":3279788608}}
```

Comment:
don't commit this

Reply:
Thank you, i missed this one during rebase.